src/appleseed/foundation/math/bvh/bvh_tree.h

```diff
@@ -58,6 +58,9 @@ class Tree
     // Return the size (in bytes) of this object in memory.
     size_t get_memory_size() const;
 
+    // Return the nodes in the tree.
+    const NodeVector& get_nodes() const;
```

Comment:
Is suggest adding `is_empty` instead.

Reply:
And i think you don't use it anymore do you ? If not, remove this

Reply:
Nice catch, we don't want it anymore.